src/frontend/src/app/app.component.scss

```diff
@@ -1,4 +1 @@
-.mat-toolbar {
```

Comment:
removing this affected the toolbar in the problems page, we can move this code into that css file.

![image](https://user-images.githubusercontent.com/57273388/211221893-e9e830ef-b30d-400d-b354-296ae8501ff4.png)